pkg/pillar/cmd/zedmanager/updatestatus.go

```diff
@@ -258,6 +259,52 @@ func updateOrRemove(ctx *zedmanagerContext, status types.AppInstanceStatus) {
 	}
 }
 
+func checkDiskSize(ctxPtr *zedmanagerContext,
```

Comment:
This function seems to overload the err return with internal errors (can't determine some size) with a (boolean) return that it doesn't fit.
That is very confusing. I don't see why a new app instance should fail because some internal error prevents you from determining the size of /persist.

Also, from the function's lack of comments and usage, it is not at all clear whether the function includes the to be added AppInstanceStatus. In fact, the function doesn't even use the status argument other than in a log.
```suggestion
func checkDiskSize(ctxPtr *zedmanagerContext,
```

Reply:
Agreed.. Initially I was excluding the specific app.. Fixed it.. Now there is no reason to pass the specific app ( status ). Fixed it.

The purpose of the function is to get the total size of disk used by all the apps on the device.